### DISCUSSION TITLE
Relay based on "From:" address

### DISCUSSION THREAD

**Discussion by gil04** [Mailu/Mailu#3177] [User support]
Hello,
I would like to use a relay host for some emails based on the from address. Is this possible?
Thx!

**Comment by ghostwheel42** [comment]
You should be able to do this using the sender_dependent_relayhost_maps config via an override for postfix. 

Alex

**Comment by gil04** [comment]
Hi Alex,

thx for your anser:

I tried this:
overrides/postfix/postfix.cf:
`sender_dependent_relayhost_maps = texthash:/overrides/relay_hosts`

overrides/postfix/relay_hosts:
`senderemail@example.com	[smtp.externalserver.com]:587`

But it still uses the MX Record to resolv the server.

Could it be that the SRS from rewriting is the problem that the from address does not match the email adress configured in relay_hosts? Has someone a workaround for this?

Thx

**Comment by ghostwheel42** [comment]
Sorry, but no, I have no idea. I was just citing the postfix manual. I looks correct to me, though.
But I didn't use the overrides yet, did you do it as described in the manual for yout mailu version?
